src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -162,7 +161,7 @@ def _stream_logs(byte_size,  # pylint: disable=too-many-locals, too-many-stateme
         # modified date and the last modified date has timed out, exit
         if ((last_modified is not None and _blob_is_not_complete(metadata)) or start < available):
 
-            delta = datetime.utcnow().replace(tzinfo=pytz.utc) - last_modified
```

Comment:
@djyou, FYI, this is an experimental change to see whether we are able tp pull out this expensive package and replace with less heavy ones. I will not merge this PR w/o going through the change with you.

Reply:
Thank you. /cc @ehotinger @northtyphoon

Reply:
iirc this LOC uses timezone aware and timezone naive objects, they added support for `.now(timezone.utc)` in 3.2 but I don't believe an equivalent existed in 2.x, hence the lib. Anywho, I'm pretty sure this will break the code, we can test tomorrow if need be.

Reply:
@ehotinger, thanks for the comments.
For context, the focus here is on CLI's windows msi installer which works around Python 3.6. 
Even Python 2.7 still needs `pytz`, but for Python 3.2+ we can live w/o it which would benefit the msi installer.
Out of the context of MSI installer, CLI will continue to work for Python 2.7 and 3.5+ both